apps/admin-gui/src/app/shared/components/dialogs/add-member-dialog/add-member-dialog.component.html

```diff
@@ -1,80 +1,91 @@
 
-  <div mat-dialog-content *ngIf="!processing">
-    <mat-form-field class="mr-2 search-field adjust-width">
-      <input
-        matInput
-        autocomplete="false"
+  <div *ngIf="failed.length === 0">
+    <div mat-dialog-content>
+      <perun-web-apps-debounce-filter
+        error="{{'DIALOGS.ADD_MEMBERS.EMPTY_SEARCH_MESSAGE' | translate}}"
         placeholder="{{'DIALOGS.ADD_MEMBERS.DESCRIPTION' | translate}}"
-        data-cy="search-members"
-        (keyup.enter)="onSearchByString()"
-        [formControl]="searchCtrl" />
-      <mat-error>
-        {{'DIALOGS.ADD_MEMBERS.EMPTY_SEARCH_MESSAGE' | translate}}
-      </mat-error>
-    </mat-form-field>
+        [autoFocus]="true"
+        [control]="searchCtrl"
+        (filter)="search.emit($event)"></perun-web-apps-debounce-filter>
 
-    <button
-      mat-flat-button
-      class="ml-auto"
-      color="accent"
-      data-cy="search-button"
-      [disabled]="loading"
-      (click)="onSearchByString()">
-      {{'DIALOGS.ADD_MEMBERS.SEARCH' | translate}}
-    </button>
-
-    <div class="no-bounce-scrollbar">
-      <mat-spinner *ngIf="loading" class="mr-auto ml-auto"></mat-spinner>
-    </div>
-    <div class="mt-3" *ngIf="!!members && !loading">
-      <app-members-candidates-list
-        #list
-        [group]="data.group"
-        [tableId]="tableId"
-        [members]="members"
-        [selection]="selection"
-        [type]="data.type"
-        [blockManualAdding]="manualAddingBlocked"
-        *ngIf="firstSearchDone">
-      </app-members-candidates-list>
+      <div class="no-bounce-scrollbar">
+        <mat-spinner *ngIf="members === null" class="mr-auto ml-auto"></mat-spinner>
+      </div>
+      <div class="mt-3" *ngIf="!!members && !loading">
+        <app-members-candidates-list
+          [tableId]="tableId"
+          [members]="members"
+          [selection]="selection"
+          [blockManualAdding]="manualAddingBlocked">
+        </app-members-candidates-list>
+      </div>
     </div>
-  </div>
-  <mat-spinner class="mr-auto ml-auto" *ngIf="processing"></mat-spinner>
-  <div mat-dialog-actions *ngIf="!processing">
-    <button mat-flat-button class="ml-auto" (click)="onCancel()">
-      {{'DIALOGS.ADD_MEMBERS.CANCEL' | translate}}
-    </button>
-    <span
-      matTooltip="{{'DIALOGS.ADD_MEMBERS.ADD_MEMBER_PERMISSION_TOOLTIP' | translate}}"
-      matTooltipPosition="above"
-      [matTooltipDisabled]="selection.selected.length === 0 || this.data.type !== 'group' || (list !== undefined && list.addAuth)">
+    <div mat-dialog-actions>
+      <button mat-flat-button class="ml-auto" (click)="cancel.emit()">
+        {{'DIALOGS.ADD_MEMBERS.CANCEL' | translate}}
+      </button>
+      <span
+        matTooltip="{{'DIALOGS.ADD_MEMBERS.ADD_MEMBER_PERMISSION_TOOLTIP' | translate}}"
+        matTooltipPosition="above"
+        [matTooltipDisabled]="selection.selected.length === 0 || addAuth">
```

Comment:
```suggestion
        [matTooltipDisabled]="selection.selected.length === 0 || inviteAuth">
```
this is tooltip for invite button